model/gasmaleperf.py

```diff
@@ -350,28 +350,21 @@ def setup(self, wind=False, DF70=True):
     def process_result(self, sol):
         print sol("MTOW")
 
+
 def test():
-    M = Mission()
-    M.cost = 1/M["t_Mission, Loiter"]
-    subs = {"b_Mission, Aircraft, Wing": 24,
-            "l_Mission, Aircraft, Empennage, TailBoom": 7.0,
-            "AR_v": 1.5, "AR": 24, "SM_{corr}": 0.5, "AR_h": 4, "k": 0.0,
-            "(1-k/2)": 1, "d_0": 1}
-    M.substitutions.update(subs)
-    for p in M.varkeys["P_{avn}"]:
-        M.substitutions.update({p: 65})
-    for t in M.varkeys["\\theta_{max}"]:
-        M.substitutions.update({t: 65})
-    M.localsolve()
```

Comment:
@mjburton11, please confirm that the substitutions I deleted here are superseded by those now in the `solve_jho` method. I copied those from the old `if __name__ == "__main__"` section.